tools/Makefile.am

```diff
@@ -11,3 +11,9 @@ dhtchat_LDFLAGS = -lopendht -lreadline -L@top_builddir@/src/.libs @Argon2_LDFLAG
 
 dhtscanner_SOURCES = dhtscanner.cpp
 dhtscanner_LDFLAGS = -lopendht -lreadline -L@top_builddir@/src/.libs @Argon2_LDFLAGS@ @GnuTLS_LIBS@
+
+if ENABLE_PROXY_SERVER
+dhtnode_LDFLAGS += -lrestbed -ljsoncpp
```

Comment:
These are private dependencies (restbed and jsoncpp are not part of the public API/ABI), this means that the OpenDHT library shouldn't have any missing symbols from those dependencies (link them properly when building the library).

Reply:
Done